src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1351,6 +1351,28 @@ public function findTaggedServiceIds(string $name, bool $throwOnAbstract = false
         return $tags;
     }
 
+    /**
+     * Finds definitions of value-objects by tag.
+     *
+     * @return array<string, array> An array of tags with the tagged class as key, holding a list of attribute arrays
+     */
+    public function findTaggedValueObjects(string $name, bool $autoExclude = true): array
```

Comment:
Could you add a phpdoc for the `autoExclude` param maybe, especially since this is public API? I kind of find it hard to get its purpose at first sight